ee_tests/src/support/core.ts

```diff
@@ -523,3 +523,20 @@ export class ScreenshotManager {
 
 export let screenshotManager: ScreenshotManager = new ScreenshotManager();
 
+export class WindowManager {
+
+  private windowCount = 1;
+
+  async switchToMainWindow() {
+    let handles = await browser.getAllWindowHandles();
+    await browser.switchTo().window(handles[0]);
+  }
+
+  async switchToNewWindow() {
+    this.windowCount++;
+    switchToWindow(this.windowCount, this.windowCount - 1);
```

Comment:
Why is `switchToWindow()` implemented outside of `WindowManager`?

Reply:
That was the previous implementation that is still used by booster tests.